packages/scripts/src/create-entrypoint.ts

```diff
@@ -109,6 +110,7 @@ export function createMainEntrypoint({
     externalFeaturesRoute,
     eagerEntrypoint,
     featuresBundleName,
+    configLoader = '@wixc3/config-loader'
```

Comment:
if this is not in a separate package, you could just pass `join(__dirname, 'default-config-loader')`. The logic will work the same, no?

Reply:
also I think about the naming. configLoader implies to me a method or something.
Maybe we should consider something like `configLoaderModulePath`?